code/modules/mob/living/silicon/ai/life.dm

```diff
@@ -158,6 +158,8 @@
 			health = 100 - getOxyLoss() - getToxLoss() - getFireLoss() - getBruteLoss()
 		diag_hud_set_status()
 		diag_hud_set_health()
+		ear_damage = null

```

Comment:
@tigercat2000 ...a lot off vars...like weakened, dizzines, etc...are all null by default, not 0.

I have no idea why.